### PR TITLE
Adds gender changing pill.

### PR DESCRIPTION
**Pull request by Supermichael777**
I'm considering adding anti borer but think that it may be use as a validing tool insted of truest ARR-PEE.

:cl: NT department of gender studies
add: Added the red-pill and a recipe.  It swaps your gender and cannot be used on others.
/:cl:

> We don't need memes at this time
Leave it for a couple of weeks.

Now literally volunteer only. 

AHHHHHHHHH!

And 100% less clinical rape
Better living through chemistry

Note this does affect your dna so it is PERMANENT THROUGH CLONING AND GENETICS.

### CODE REVIEW

**Comment by optimumtact** [comment]
For what purpose or reason are you adding this?

**Comment by ShadowDeath6** [comment]
this can be used to meta whether or not they were male or female before being redpilled, hmmm....

**Comment by Thunder12345** [comment]
@optimumtact memes

**Comment by anconfuzedrock** [comment]
you realise this is HARDER to make than the one in xenobio that works on others instantly, right? speaking of which you can literally run up to somebody and click them with the potion

**Comment by ExcessiveUseOfCobblestone** [comment]
The joke about the gender surgery argument is that no one actually ever used the surgery.

File: code/modules/reagents/reagent_containers/pill.dm [hunk 1]
```diff
@@ -142,3 +142,33 @@
...
+	if(M == user)
+		if(M.gender == FEMALE)
+			M.gender = MALE
+			newgender = pick(dude, man, boy, bro)
+		else
+			M.gender = FEMALE
+			newgender = pick(dudette, woman, girl, brodette)
+		M.regenerate_icons()
+		M.dna.generate_uni_identity()
+		M.visible_message("<span class='notice'>[user] swalows the [src] and morphs into a [newgender]!</span>")
```
Comment (coiax) [inline code/modules/reagents/reagent_containers/pill.dm]: swalows

**Comment by optimumtact** [comment]
This already exists through slime potions and you haven't justified why you think we need another method

**Comment by Supermichael777** [comment]
>xenobiology ever doing their job.

This restores it to where it rightfully should be: med-bay

**Comment by optimumtact** [comment]
But it's not medbay, it's chemistry

**Comment by Supermichael777** [comment]
Chemistry is a part of medbay no mater what the chemist says
1. chemistry has a crew facing window and can  whip this shit up quick/provide that one missing ingredient.
2 xenobio is an armored bunker isolated by at least 3 airlocks and a long hallway and it takes 10~ minnutes unless they already had it.

**Comment by optimumtact** [comment]
So basically you're dumbing the game down by making it easier to do something that was previously requiring a little bit of effort and skill?

**Comment by Supermichael777** [comment]
No.

 I'm making an obscure cure for a isolated condition that barley affects game-play but makes people feel uncomfortable easier to access for people who aren't going to grief with it.  This can be safely handed out to any old shitler.  This still requires quite a bit of processing for a single use cosmetic. I wouldn't say its easier or safer, setting off teslium by mistake is fairly deadly.

**Comment by bgobandit** [comment]
this only exists because of a shitty meme

**Comment by ShadowDeath6** [comment]
also is there a reason it's not actually a mutation toxin

File: code/modules/reagents/reagent_containers/pill.dm [hunk 2]
```diff
@@ -142,3 +142,38 @@
...
+			H.dna.update_ui_block(DNA_GENDER_BLOCK)
+			H.update_body()
+			H.update_mutations_overlay()
+			H.visible_message("<span class='notice'>[user] swallows the [src] and morphs into a [newgender]!</span>")
+			qdel(src)
+		else
+			user.visible_message("<span class='notice'>[user] swallows the [src] but isnt complex enough to understand gender issues!</span>")
+	else
+		M.visible_message("<span class='notice'>[user] attempts to feed [M] the [src] but it turns blue!  Looks like [M] wasn't ready to see from the other side.</span>")
+		qdel(src)
```
Comment (ShadowDeath6) [inline code/modules/reagents/reagent_containers/pill.dm]: why does it delete itself if you try to feed it to someone else, just have it not work

Comment (ExcessiveUseOfCobblestone) [inline code/modules/reagents/reagent_containers/pill.dm]: along with make it blue to fit the alleged lore

Comment (Supermichael777) [inline code/modules/reagents/reagent_containers/pill.dm]: its to prevent spamming

Comment (ShadowDeath6) [inline code/modules/reagents/reagent_containers/pill.dm]: how would this spam, this is the else to the `M == user` check

**Comment by optimumtact** [comment]
Okay so to even get this considered to be merged you have to make it an actual reagent pill

this random snowflake item will not pass go and will not receive 200 dollars

**Review by Cyberboss — CHANGES_REQUESTED** [review]
I have no comment on the feature itself

But, fuuuuuck your implementation! Make it a proper reaction of the horomone reagent

**Comment by Supermichael777** [comment]
Well seeing as the whole point of it not being a chemical was to prevent turbonerds from using for griff.

**Comment by ShadowDeath6** [comment]
how is making it a chemical going to make it grief usable

**Pull request closed without merging**